### ISSUE TITLE
Profiler cannot do Advanced Rocketry dimensions.

### ISSUE THREAD

**Issue by IMarvinTPA** [way2muchnoise/JustEnoughResources#134]
The profiler does not visit any planet dimensions from Advanced Rocketry.

This bit of error messages shows up when I try to profile a planet while on it:
```
[03:19:35] [pool-22-thread-1/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: java.lang.NullPointerException
[03:19:35] [pool-22-thread-1/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at net.minecraftforge.common.DimensionManager.getProvider(DimensionManager.java:153)
[03:19:35] [pool-22-thread-1/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at jeresources.profiling.ProfilingTimer.getDimensionName(ProfilingTimer.java:84)
[03:19:35] [pool-22-thread-1/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at jeresources.profiling.ProfilingTimer.startChunk(ProfilingTimer.java:34)
[03:19:35] [pool-22-thread-1/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at jeresources.profiling.ChunkProfiler.profileChunk(ChunkProfiler.java:46)
[03:19:35] [pool-22-thread-1/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at java.util.ArrayList.forEach(ArrayList.java:1255)
[03:19:35] [pool-22-thread-1/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at jeresources.profiling.ChunkProfiler.run(ChunkProfiler.java:41)
[03:19:35] [pool-22-thread-1/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[03:19:35] [pool-22-thread-1/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[03:19:35] [pool-22-thread-1/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at java.lang.Thread.run(Thread.java:748)
[03:19:39] [pool-22-thread-4/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: java.lang.NullPointerException
[03:19:42] [pool-22-thread-7/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: java.lang.NullPointerException
```

Maybe have an option to have it profile the real chunks?

Thanks,
IMarv

**Comment by IMarvinTPA** [comment]
This issue appears to be caused by JER confusing dimension type IDs with dimension IDs.  Please see research in the linked AR issue.

IMarv

**Comment by IMarvinTPA** [comment]
This may help: https://github.com/way2muchnoise/JustEnoughResources/pull/138

**Comment by way2muchnoise** [comment]
Fixed in #138
